README.md

```diff
@@ -124,11 +124,12 @@ This library aims to support and is [tested against][travis] the following Ruby
 implementations:
 
 * Ruby 1.9.3
-* Ruby 2.0.0
+* Ruby 2.0
 * Ruby 2.1
 * Ruby 2.2
 * Ruby 2.3
 * Ruby 2.4
+* Ruby 2.5
 * [JRuby 9K][jruby]
```

Comment:
We actually still support JRuby 1.7 in this release as well.  May as well list it too.